bftengine/include/bftengine/ClientMsgs.hpp

```diff
@@ -39,8 +40,16 @@ struct ClientReplyMsgHeader {
   uint32_t spanContextSize = 0u;
   uint16_t currentPrimaryId;
   uint64_t reqSeqNum;
+
+  // Reply length is the total length of the reply, including any replica specific info.
   uint32_t replyLength;
+
+  // This is the size of the replica specific information. If it is 0, there is no replica specific
+  // information. The offset of the replica specific information from the start of the reply message
+  // is `replyLength - replicaSpecificInfoLength`.
+  uint32_t replicaSpecificInfoLength = 0;
```

Comment:
I would prefer to see replicaSpecificInfo instead of rsi :-)

Reply:
I don't follow. Isn't that what's written?

Reply:
You used rsi in Reply and all other places till now.

Reply:
Ah, yeah. I prefer the shorter form for most things, since it just makes the calling code shorter. I'm open to changing in another PR though.  I used the longer form in the message to match the existing style, and also because it's only used in one place in the client :)